### ISSUE TITLE
Treatment endtime is not avaliable?

### ISSUE THREAD

**Issue by peiyaoli** [MIT-LCP/eicu-website#36]
It seems that endtime of treatment is not available here? So how could we find the treatment duration? Such as duration of CRRT? @tompollard @alistairewj

**Comment by rincont** [comment]
I agree that this is not trivial - different treatments would require different end times. For example CRRT calculations may be required as frequently as 100 to 160 times in a 24 hour period and nurse are documenting hourly so an end time of 16 hours without documentation is not meaningful.

**Comment by peiyaoli** [comment]
Could you please to give us a demo? @alistairewj

**Comment by alistairewj** [comment]
Not sure I'll have time to look at this in the next few weeks. @tompollard
will be releasing an intake output table soon and that might have some data
elements to look at. Also we should probably move this discussion to
eicu-code as it's not about documentation of the database.
